modules/sql-engine/src/integrationTest/sql/cast/test_cast_interval_day.test

```diff
@@ -0,0 +1,144 @@
+# name: test/sql/cast/test_cast_interval_day.test
```

Comment:
let's add a few test cases for complex intervals (like DAYS TO MINUTES). The same for YEAR-MONTH intervals